examples/oss-na-25-demo/README.md

```diff
@@ -0,0 +1,80 @@
+# Open Source Summit North America (OSS NA) '25 Provenance Demo
+
+## Introduction
+
+This example demonstrates how to generate and verify provenance data for a
```

Comment:
We should streamline some of these README files since they are very similar.

Reply:
tracking this in #48